spec.md

```diff
@@ -210,6 +216,17 @@ and the event data will be materialized. For example, in the case of a JSON
 serialization, the context attributes and the event data might both appear
 within the same JSON object.
 
+Context attributes MUST be one of the following types:
```

Comment:
I believe the half sentence in line 227 is sufficient for this PR.

`Map` values MUST NOT be used in context attributes, including in extension attributes.

Reply:
Done. I'm slightly uncomfortable with the formulation, but the result is clear if you read the whole spec:

> The value of a context attribute is Integer, String, Binary, Map, URI-reference, Timestamp, or Any, except that it can't be Any or Map.

If we accept some form of #457, we can completely remove Map, and the sentence simplifies to:

> The value of a context attribute is Integer, String, Binary, Map, URI-reference, Timestamp, or Any.